packages/react-components/react-motion/library/src/hooks/useAnimateAtoms.ts

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import type { AnimationHandle, AtomMotion } from '../types';
 
+// eslint-disable-next-line no-restricted-globals
+const win = typeof window !== undefined ? window : undefined;
```

Comment:
We also have a `canUseDOM` utility that checks `typeof window !== 'undefined' && !!(window.document && window.document.createElement)` -- might be worth using that for consistency

Reply:
e.g.

```js
const SUPPORTS_WEB_ANIMATIONS = /*@__PURE__*/ (() => canUseDOM() && typeof Element.prototype.animate === 'function')();
```

Reply:
I was using canUseDOM at first, but since we are accessing the Element global directly rather than through the window, I was planning on updating it to window.Element. This then violates the rule for window globals, so instead of using canUseDOM + disabling the lint rule, I opted to do it in a single variable like this.

Reply:
> might be worth using that for consistency

sounds like having a standalone package to provide these checks for core library might be time well spent ! ;)  a tiny in house version of https://modernizr.com/ ( aaah old good times :D )